examples/with-chakra-ui-typescript/package.json

```diff
@@ -3,15 +3,15 @@
   "scripts": {
     "dev": "next",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "lint": "next lint"
   },
   "dependencies": {
-    "@chakra-ui/icons": "^1.0.5",
-    "@chakra-ui/react": "^1.4.2",
-    "@chakra-ui/theme-tools": "1.1.2",
-    "@emotion/react": "11.1.5",
-    "@emotion/styled": "11.1.5",
-    "framer-motion": "^4.0.3",
+    "@chakra-ui/icons": "^1.1.7",
+    "@chakra-ui/react": "1.8.8",
+    "@emotion/react": "^11",
+    "@emotion/styled": "^11",
+    "framer-motion": "^6",
     "next": "latest",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
```

Comment:
Could we update these and all other `devDependencies` as well, maybe?

Reply:
We still don't have full support for React 18 in Chakra UI. All the necessary changes will be part of v2.0, but I plan on circling back to update the examples when we release it.